docker/images/fuseki/.dockerignore

```diff
@@ -0,0 +1 @@
+t11
```

Comment:
In gitmodules you added t11 as a submodule, but in dockerignore you have it set to exclude this dir.